api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -44,7 +44,7 @@
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
   public ResponseEntity<Void> updateUserDisabledStatus(UpdateUserDisabledRequest request) {
```

Comment:
It would be nice to have a follow up PR to update the field in UpdateUserDisabledRequest as well to username to be consistent

Reply:
Good catch. Let's do that with the UI changes.